migration/migration.go

```diff
@@ -229,6 +229,7 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr *models.GormWork
 	if err := createOrUpdatePlannerItemExtention(models.SystemBug, ctx, witr, db); err != nil {
 		return err
 	}
+	witr.ClearCache() // Clear the WIT cache after updating existing WITs
```

Comment:
Why not just the put the new version of a type into the cache when we update it? That would work even when we load a type more then once in the update.